hw1/tasks/task03.py

```diff
@@ -0,0 +1,26 @@
+
+"""
+Write down the function, which reads input line-by-line, and find maximum and minimum values.
+Function should return a tuple with the max and min values.
+For example for [1, 2, 3, 4, 5], function should return [1, 5]
+We guarantee, that file exists and contains line-delimited integers.
+To read file line-by-line you can use this snippet:
+with open("some_file.txt") as fi:
+    for line in fi:
+        ...
+"""
+from typing import Tuple
+import numpy as np
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
+    with open(file_name) as fi:
+        max = -np.inf
```

Comment:
numpy is quite a huge module - it's better to avoid it for such simple function
here it can be replaced by just comparing values not to np.inf, but to one of numbers